security/pkg/caclient/clientrunner.go

```diff
@@ -0,0 +1,80 @@
+// Copyright 2018 Istio Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package caclient
+
+import (
+	"fmt"
+	"time"
+
+	"istio.io/istio/pkg/log"
+	pkiutil "istio.io/istio/security/pkg/pki/util"
+	"istio.io/istio/security/pkg/util"
+)
+
+// ClientRunner interacts with the upstream CA to maintain the KeyCertBundle valid.
```

Comment:
Naming it CertRefresher/Refresher maybe more informative as that's what it actually does.

Reply:
With CertRefresher { Start/Run(), Stop()} I think it gives the user some rough idea. We can change Run/Start to 
StartLoop or something if want to emphasize the continuously running behavior.

The library runs CAClient to do something. I just feel we should name by what rather than how/implementation. Otherwise lib user has to rely on doc string to understand what it does.